src/app/investment-details/investment-details.component.scss

```diff
@@ -1,127 +0,0 @@
-@import "app";
-
-::ng-deep {
```

Comment:
as you see in `src/app/investment-details/investment-details.component.html`, there is no markup to style so this CSS should belong to offer details screen because, as we discussed earlier, everything is bloated inside this component (we'll refactor it later). there is no need for ng-deep here. I moved it to offer details.